FE403/index.js

```diff
@@ -0,0 +1,94 @@
+const cards = document.querySelectorAll('.card');
+
+let hasFlippedCard = false;
+let firstCard, secondCard;
+let matchedCard = document.getElementsByClassName("match");
+let restart_btn = document.getElementById('restart');
+
+function shuffleDeck() {
+    cards.forEach(card =>
+    {
+        var randomNumber = Math.floor(Math.random() * 16);
+        card.style.order = randomNumber;
+    });
+}
+
+function flip() {
+    if (this === firstCard) return;
+  
+    this.classList.add('flipped');
+  
+    if (!hasFlippedCard) {
+        hasFlippedCard = true;
+        firstCard = this;
+        console.log("first:" + firstCard.getAttribute('data-id'));
+        return;
+    }
+    
+    secondCard = this;
+    console.log("second:" + secondCard.getAttribute('data-id'));
+    ifMatch();
```

Comment:
Our standard practice for functions is to give 'em names that start with verbs. That way, it's easier to read and make sense of (especially if you're working with other developers).